backend/manager/modules/restapi/jaxrs/src/main/java/org/ovirt/engine/api/restapi/resource/BackendDiskSnapshotsResource.java

```diff
@@ -36,13 +36,18 @@ public DiskSnapshots list() {
     protected DiskSnapshots mapCollection(List<org.ovirt.engine.core.common.businessentities.storage.Disk> entities) {
         DiskSnapshots collection = new DiskSnapshots();
         for (org.ovirt.engine.core.common.businessentities.storage.Disk disk : entities) {
-            DiskSnapshot diskSnapshot = getMapper(org.ovirt.engine.core.common.businessentities.storage.Disk.class, DiskSnapshot.class).map(disk, null);
+            DiskSnapshot diskSnapshot =
+                    getMapper(org.ovirt.engine.core.common.businessentities.storage.Disk.class, DiskSnapshot.class).map(disk, null);
```

Comment:
If I'm not mistake, you just tried to fix the long line here, but the new line is still too long - 133 characters.
So better either to revert or split to one more line, i.e., to break `.map(disk, null);` to a new line.